src/coverlet.collector/InProcDataCollection/CoverletInProcDataCollector.cs

```diff
@@ -12,8 +11,11 @@ namespace Coverlet.Collector.DataCollection
 {
     public class CoverletInProcDataCollector : InProcDataCollection
     {
+        private readonly TestPlatformEqtTrace _eqtTrace = new TestPlatformEqtTrace();
```

Comment:
Why not instantiate this in the `Initialize` method? Will the method be called more than once?

Reply:
Actually no I'll move to init so we won't pay for the instance if not used